src/auth/digest/Config.h

```diff
@@ -73,15 +91,12 @@ class digest_nonce_h : public hash_link, public RefCountable
 
     /* has this nonce been invalidated ? */
     struct {
-        bool valid = true;
+        mutable bool valid = true;
```

Comment:
Can we undo this change? Is there some code that modifies this flag in a constant object?

I did suggest making this flag mutable in a change request, but only in conjunction with making valid() constant. In the current PR code, there are only non-constant validate() methods. They do not need this flag to be mutable.